src/core/components/AMInstallButton/index.js

```diff
@@ -286,6 +287,10 @@ export class AMInstallButtonBase extends React.Component<InternalProps> {
       userAgentInfo,
     } = this.props;
 
+    if (!isFirefox({ userAgentInfo })) {
+      return null;
```

Comment:
Maybe we should add a `_log.debug()`, or maybe not. I don't know.

Reply:
I don't think so. This wouldn't be unexpected. I am adopting a strategy of always rendering the `AMInstallButton` and letting it determine whether to render anything or not, rather than forcing the parent to figure out whether or not to render the `AMInstallButton`. Therefore, it could be quite frequent that this returns `null` (i.e., anytime a non-Firefox browser visits the page).